README.md

```diff
@@ -16,10 +15,8 @@ data.
 
 ## Installation
 
```

Comment:
Why did you remove the code block on installation code?

Reply:
good question that was not intentional. looking at the commit history for README.Rmd it looks like the last commit that made changes to it was by Evan on 2023-10-23.

Reply:
great question that was not intentional.

Reply:
I think that it is a just an updated syntax? those changes occured when I rendered the .md file from the Rmd file and it is still showing up as code block in the README section on the gh site.